Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/RoomWrapper.java

```diff
@@ -0,0 +1,261 @@
+package nl.soccar.gameserver.model.session;
+
+import nl.soccar.gameserver.GameServerConstants;
+import nl.soccar.gameserver.controller.rmi.GameServerRmiController;
+import nl.soccar.gameserver.controller.socnet.message.*;
+import nl.soccar.gameserver.model.GameServer;
+import nl.soccar.gameserver.model.PlayerWrapper;
+import nl.soccar.gameserver.model.command.Command;
+import nl.soccar.gameserver.model.command.CommandDispatcher;
+import nl.soccar.gameserver.util.RoomUtilities;
+import nl.soccar.library.Player;
+import nl.soccar.library.Room;
+import nl.soccar.library.Team;
+import nl.soccar.library.enumeration.Privilege;
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.socnet.connection.Connection;
+
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Lesley
+ */
+public final class RoomWrapper {
+
+    private static final Logger LOGGER = Logger.getLogger(RoomWrapper.class.getSimpleName());
+    private static final Random RANDOM = new Random();
+
+    private final List<PlayerWrapper> players = new ArrayList<>();
+    private final Map<PlayerWrapper, Team> teams = new HashMap<>();
+
+    private final List<String> mutedNames = new ArrayList<>();
+
+    private final SessionWrapper session;
+    private final Room room;
+
+    public RoomWrapper(SessionWrapper session, Room room) {
+        this.session = session;
+        this.room = room;
+    }
+
+    public void join(PlayerWrapper player, String password) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)